internal/cmd/local/command_service.go

```diff
@@ -0,0 +1,765 @@
+package local
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/hashicorp/go-version"
+	"github.com/spf13/cobra"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+
+	"github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/spinner"
+)
+
+func NewServiceCommand(service string, prerunner cmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   service,
+		Short: fmt.Sprintf("Manage %s.", writeOfficialServiceName(service)),
+		Args:  cobra.NoArgs,
+	}
+
+	switch service {
+	case "zookeeper":
+		cmd.Aliases = []string{"zk"}
+	case "schema-registry":
+		cmd.Aliases = []string{"sr"}
+	}
+
+	c := NewLocalCommand(cmd, prerunner)
+
+	c.AddCommand(NewServiceLogCommand(service, prerunner))
+	c.AddCommand(NewServiceStartCommand(service, prerunner))
+	c.AddCommand(NewServiceStatusCommand(service, prerunner))
+	c.AddCommand(NewServiceStopCommand(service, prerunner))
+	c.AddCommand(NewServiceTopCommand(service, prerunner))
+	c.AddCommand(NewServiceVersionCommand(service, prerunner))
+
+	switch service {
+	case "connect":
+		c.AddCommand(NewConnectConnectorCommand(prerunner))
+		c.AddCommand(NewConnectPluginCommand(prerunner))
+	case "kafka":
+		c.AddCommand(NewKafkaConsumeCommand(prerunner))
+		c.AddCommand(NewKafkaProduceCommand(prerunner))
+	case "schema-registry":
+		c.AddCommand(NewSchemaRegistryACLCommand(prerunner))
+	}
+
+	return c.Command
```

Comment:
Do we need `c.Hidden = true` on this command and/or its subcommands? I see it on `current` `destroy` and `version` but I didn't see it here.

Reply:
it's a child cmd of services so it's already hidden

Reply:
The highlighting is kind of misleading, I was wondering if services is hidden, not schema registry acl.

Reply:
hh yes, service is a child of services, so both hidden

Reply:
Oh, I see, I misunderstood!